### DISCUSSION TITLE
[Kanban] An easier method of adding pre-existing tiddler to boards/columns

### DISCUSSION THREAD

**Discussion by AshRubigo** [BurningTreeC/tiddlywiki-muuri#17] [Ideas]
Currently to add a pre-existing tiddler to a kanban board/column, the board/column tiddler must be edited outside the kanban board interface, and then the title of the to-be-added tiddler pasted to the list field.

It would be more convenient if a tiddler link could be dragged onto the board/column.

**Comment by BurningTreeC** [comment]
Hey @AshRubigo ,

currently one can already add tiddlers to kanban columns by dropping a tiddler link onto the column header

**Reply by AshRubigo** [reply]
I see. This wasn't clear because that is different to the functionality of dragging cards. Perhaps highlighting the header of columns while dragging a tiddler link could improve the intuitiveness of that feature.

**Reply by BurningTreeC** [reply]
Yes @AshRubigo that's true. I'll see if I can do that
